src/redux/book/bookSlice.jsx

```diff
@@ -0,0 +1,28 @@
+import {createSlice} from "@reduxjs/toolkit"
+
+
+const initialState = {
+  bookStore :[],
+  title:"lastNight",
+  author:"miki",
+  isLoading:true
+}
+
+const bookSlice = createSlice({
+  name:"books",
+  initialState,
+  reducers :{
+    addBook : () =>{
+   
+    },
```

Comment:
- [ ]  Great job to implement redux configurations and the required methods but your `addbook `method is empty now.  Please consider the state changes while user add a book and add the logic to implement that as well.